AnnoyancesFilter/sections/popups.txt

```diff
@@ -129,6 +129,8 @@ xn----7sba7aachdbqfnhtigrl.xn--j1amh#$#div[id^="pum-"][data-popmake*="nastrojka-
 xn----7sba7aachdbqfnhtigrl.xn--j1amh#$#html { overflow: visible!important; }
 backblaze.com##.ju_Con
 honeysanime.com##.buruburu-bee
+postandcourier.com#?#.tp-modal:has(> div.tp-iframe-wrapper iframe[src^="https://buy.tinypass.com/checkout/offer/show?displayMode"])
+postandcourier.com#$#.tp-backdrop { display: none !important; }
 postandcourier.com#%#//scriptlet('remove-class', 'tp-modal-open', 'body')
```

Comment:
Scriptlet rule is good here, but try also this `postandcourier.com#$#html > body.tp-modal-open { overflow: auto !important; }` - it should work fine with the app.